src/api_types.jl

```diff
@@ -23,11 +23,11 @@ end
 const HDF5ReferenceObj_NULL = HDF5ReferenceObj(UInt64(0))
 
 # For group information
-struct H5Ginfo
-    storage_type::Cint
+struct H5G_info_t
+    storage_type::Cint # enum H5G_storage_type_t
     nlinks::hsize_t
     max_corder::Int64
-    mounted::Cint
+    mounted::Cuint
```

Comment:
I think we should just add `const hbool_t = Cuint` and use that for the type of mounted.

Reply:
:+1: I actually went back and forth a couple of times on whether to add that or not, so glad you think we should since I initially wanted to.